README.md

```diff
@@ -26,20 +26,22 @@ The following configuration options are optional.
 | CALLER_KEY            | The caller key that WiseTime should provide with post time webhook calls. The connector does not authenticate Webhook calls if not set. |
 | TAG_UPSERT_PATH       | The WiseTime tag folder path to use for Jira tags. Defaults to `/Jira/` (trailing slash is required). Use `/` for root folder.          |
 | TAG_UPSERT_BATCH_SIZE | Number of tags to upsert at a time. A large batch size mitigates API call latency. Defaults to 500.                                     |
-| PROJECT_KEYS_FILTER   | If set, the connector will only handle Jira issues from the configured Jira project keys.                                               | 
+| PROJECT_KEYS_FILTER   | If set, the connector will only handle Jira issues from the configured Jira project keys.                                               |
+| PERSISTENT_DIR        | If set, the connector will use the directory as the storage location for SQLite to keep track the Jira issues it have synced.           |
```

Comment:
Would prefer `DATA_DIR`. The directory may or may not be persistent. As far as connector is concerned, it's just a dir.

Reply:
Ok makes sense. But just to clarify, this is implemented on the [connector library](https://github.com/wisetime-io/wisetime-connector-java/blob/master/src/main/java/io/wisetime/connector/config/ConnectorConfigKey.java#L14) and is currently just used for the connector store.
Should I update the connector library?

Reply:
Yes please